<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetVpzPropertyDynamics</class>
 <widget class="QWidget" name="WidgetVpzPropertyDynamics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>108</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QLabel" name="labelName">
     <property name="text">
      <string>&lt;none&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelPackage">
     <property name="text">
      <string>&lt;noPackageSet&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelLibrary">
     <property name="text">
      <string>&lt;noLibrarySet&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
